cpp/src/arrow/adapters/orc/adapter_util.h

```diff
@@ -34,8 +34,17 @@ namespace orc {
 
 Status GetArrowType(const liborc::Type* type, std::shared_ptr<DataType>* out);
 
+Status GetORCType(const DataType& type, ORC_UNIQUE_PTR<liborc::Type>* out);
+
+Status GetORCType(const Schema& schema, ORC_UNIQUE_PTR<liborc::Type>* out);
+
 Status AppendBatch(const liborc::Type* type, liborc::ColumnVectorBatch* batch,
-                   int64_t offset, int64_t length, ArrayBuilder* builder);
+                   int64_t offset, int64_t length, arrow::ArrayBuilder* builder);
+
+Status WriteBatch(liborc::ColumnVectorBatch* column_vector_batch,
```

Comment:
Please add documentation here.   The style used in this code base is to order parameters in the order:
1.  Input first,
2. Input output next.
3. Output variables.

Reply:
None of the symbols here is actually exposed to library users. Do we need to document these functions?

Reply:
@pitrou @emkornfield